File: JAVA/05_1Dim_array/BOJ_02_2562_최댓값.java
import java.io.IOException;
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.util.StringTokenizer;

public class Main {
    public static void main(String[] args) throws IOException {
        int j = 0, max = 0;
        int num[] = new int[9];
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringTokenizer st = new StringTokenizer(br.readLine(), " ");
        for (int i = 0; i < 9; i++) {
            num[i] = Integer.parseInt(st.nextToken());
            if (num[i] > max) {
                max = num[i];
                j = i;
            }
        }
        System.out.println(max + "\n" + (j + 1));
    }
}

File: C/05_1Dim_array/BOJ_02_2562_최댓값.c
#include <stdio.h>
main() {
	int num[9], j, max = 0;
	for (int i = 0; i < 9; i++) {
		scanf("%d", &num[i]);
		if (num[i] > max) {
			max = num[i];
			j = i;
		}
	}
	printf("%d\n%d", max, j + 1);
}